multiqc/utils/search_patterns.yaml

```diff
@@ -767,6 +767,8 @@ samtools/rmdup:
   shared: true
 sargasso:
   fn: "overall_filtering_summary.txt"
+seqwho:
+  fn: "SeqWho_call.json"
```

Comment:
Looking at the SeqWho code, the output is written to stdout and thus can have any name. I wonder if we can detect a file based on its contents?

Reply:
Perhaps this would be unique enough:

```yaml
seqwho:
  contents: '  "Per Base Seq": ['
```

Reply:
ya that should work.